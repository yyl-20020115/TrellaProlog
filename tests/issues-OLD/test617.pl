:- op(300,xfx,\\).

main :-
	read_term_from_chars("arg(1,(\\) \\\\ '', Y).", X),
	read_term_from_chars("arg(1,\\ \\\\ '', Y).", X).

:- initialization(main).
