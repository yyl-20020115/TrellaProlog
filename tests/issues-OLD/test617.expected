   error(syntax_error(operator_expected),read_term_from_chars/3).
